Get-NetView.psm1

```diff
@@ -994,22 +1052,43 @@ function MellanoxDetailPerNic {
             "$env:ProgramFiles\Mellanox\MLNX_WinOF2_Azure_HPC"
             break
         }
+        "ipoib6x.sys" {
+            "$env:ProgramFiles\Mellanox\MLNX_VPI"
+            break
+        }
+        "mlx4eth63.sys" {
+            "$env:ProgramFiles\Mellanox\MLNX_VPI"
+            break
+        }
         default {
             $msg = "Driver $driverFileName isn't supported"
             ExecControlError -OutDir $dir -Function"MellanoxDetailPerNic" -Message $msg
             return
         }
     }
 
-    $toolName = $driverFileName -replace ".sys", "Cmd"
-    $toolPath = "$driverDir\Management Tools\$toolName.exe"
+    #
+    # Execute tool
+    #
+
+    $DriverName = $( if($driverFileName -eq "Mlx5.sys" -or $driverFileName -eq "Mlnx5.sys" -or $driverFileName -eq "Mlnx5Hpc.sys") {"WinOF2"} else {"WinOF"})
```

Comment:
`if ($driverFileName -in @("Mlx5.sys", "Mlnx5.sys", "Mlnx5Hpc.sys")`

Reply:
Done